package.json

```diff
@@ -4,31 +4,17 @@
   "description": "Manipulate sentences as a unit of movement. Select, move and delete by whole sentences.",
   "main": "main.js",
   "scripts": {
-    "start": "rollup --config rollup.config.js -w",
-    "build": "rollup --config rollup.config.js --environment BUILD:production"
+    "dev": "esbuild main.ts --bundle --external:obsidian --outdir=.  --target=es2016 --format=cjs --sourcemap=inline --watch",
+    "build": "esbuild main.ts --bundle --external:obsidian --outdir=.  --target=es2016 --format=cjs"
   },
   "keywords": [],
-  "author": "Tim Hor",
+  "author": "Andrew Brown, Tim Hor",
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^18.0.0",
-    "@rollup/plugin-node-resolve": "^11.2.1",
-    "@rollup/plugin-typescript": "^8.2.1",
-    "@types/node": "^14.14.37",
-    "husky": "^7.0.2",
-    "lint-staged": "^11.2.0",
-    "obsidian": "^0.12.0",
-    "prettier": "^2.4.1",
-    "rollup": "^2.32.1",
-    "tslib": "^2.2.0",
-    "typescript": "^4.2.4"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{ts,tsx,html,css,md,json}": "prettier --write"
```

Comment:
In case this was accidental: the whole point of having the pre-commit hook is to help with consistent formatting, so best to keep it in

Reply:
Oh, sure. It's such a tiny plugin that I thought I would remove what I didn't understand. But if this makes it easier for other people, put it back.

Reply:
Yeah, when other people's editors have different settings, the diff can sometimes end up confusing to read because it becomes difficult to tell what was a formatting change vs. actual logic change 🙂